### DISCUSSION TITLE
How to increase header section (header.cshtml) height ?

### DISCUSSION THREAD

**Discussion by Allmantool** [gotenberg/gotenberg#371] [Q&A]
I try to generate a pdf from 2 seperate html pages (i.e header.cshtml + index.cshtml).
![image](https://user-images.githubusercontent.com/16443259/136812191-816ed861-a39e-4036-94ee-f0bb086e16f7.png)

It generally works fine, instead of one moment that worries me. 
I can't set an appropriate header height up.

As you can see from the attached screenshot I tried to place a table with 6 rows within the header section, but it was truncated by 4 items
![image](https://user-images.githubusercontent.com/16443259/136812368-44fe20b8-1042-47f8-9d8e-01dd49efc8cb.png)

I believe that the root cause, that Gotenberg sets the concrete max height for a header section.

I've also set margin between-top for index.cshtml, in hope that it was just overlapping, but no success

![image](https://user-images.githubusercontent.com/16443259/136812557-04fad607-0ad5-4be0-95aa-7b0987f7583f.png)

Q: I'm just curious, maybe someone has some experience in this area and can help me to increase header height to fit all related content in the header section

**Comment by gulien** [comment]
Hello @Allmantool,

Alas, Gotenberg does not have any control regarding the height of the header/footer. I guess you could play with the font-size, margins, etc. to make your content fit to a certain extend the "allowed" height? See for instance: https://github.com/gotenberg/gotenberg/discussions/343#discussioncomment-1315551

**Reply by Allmantool** [reply]
I wish it can be done in a more convenient way, but anyway thx for the help
